com.valvesoftware.Steam.json

```diff
@@ -158,7 +154,19 @@
         },
         {
             "name": "steam",
-            "no-autogen": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop",
+                "sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop",
+                "desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop",
+                "sed -i s:/usr/lib/:/app/lib/: steam",
+                "PREFIX=/app make install",
+                "ln -sf /bin/true /app/bin/steamdeps",
+                "cp /usr/bin/cmp /app/bin",
+                "mkdir -p /app/lib/i386-linux-gnu /app/lib/debug/lib/i386-linux-gnu",
+                "install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.appdata.xml",
```

Comment:
This doesn't look like simple relocation from default to simple. Could you please split the migration and any functional changes into separate commits?

Reply:
This will not break anything, `/app/share/appdata` is the legacy path, but both are supported, see:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Reply:
We're currently supporting quite old versions of Flatpak. Does this break backwards-compatibility and from which versions? Comments in code or commits about this kind of things, please.

Reply:
https://github.com/hughsie/appstream-glib/commit/74f0b2b54890662783ebef64c7f130fa2806a8c0

Reply:
Sounds too recent for us then

Reply:
If you want to port to simple, fine but this PR is not acceptable unless you split the commits. It must be possible to say based on commit history why functional changes are done.

Reply:
How is this "recent"? appstream-glib 0.6.7 was released 12 Jan 2017, the 18.08 SDK contains 0.7.10 and there are dozens of apps on Flathub already, that is this path.

Reply:
You arguing against review comments isn't really productive in getting this merged

Reply:
Please split the commit so that it can be better followed what gets replaced by what